packages/aila/src/core/chat/AilaChat.ts

```diff
@@ -218,26 +237,66 @@ export class AilaChat implements AilaChatService {
     return applicableMessages;
   }
 
-  // #TODO this is the other part of the initial lesson state setting logic
```

Comment:
Addresses this TODO about making the initial categorisation step something pluggable / specific to the type of document we are trying to create